### PR TITLE
Fixed the phenomenon of console suspended animation in web mode

### PR DESCRIPTION
**Pull request by 1439707509**
In actual use, it is found that after a long period of translation, the console will only display 'New run task xxx' continuously without any further output, and no translated pictures will be output under the result. When closing cmd and re-executing the command, everything Back to normal.
After testing, after modifying two places, this problem disappears. One is for an existing translation picture, the console outputs "existing translation picture" to avoid misunderstanding and no response. The other is to add a progress feedback and return True when skipping existing images in manga_translator.py

### CODE REVIEW

**Comment by thatDudo** [comment]
No chinese comments please

**Comment by 1439707509** [comment]
> No chinese comments please

Is that all right?

**Comment by zyddnys** [comment]
change print to logger.info

File: manga_translator/server/web_main.py [hunk 1]
```diff
@@ -187,6 +187,8 @@ async def run_async(request):
     task_id = f'{phash(img, hash_size = 16)}-{size}-{selected_translator}-{target_language}-{detector}-{direction}'
     print(f'New `run` task {task_id}')
     if os.path.exists(f'result/{task_id}/final.jpg') or os.path.exists(f'result/{task_id}/final.png') or os.path.exists(f'result/{task_id}/final.webp'):
+        # Add a console output prompt to avoid the console from appearing to be stuck without execution when the translated image is hit consecutively.
+        print(f'[succeed] The task {task_id} has exists')
```
Comment (thatDudo) [inline manga_translator/server/web_main.py:191]: Change to print("Using cached result for {task_id}")

**Comment by thatDudo** [comment]
> change print to logger.info

We dont have logging in web_main yet.

**Comment by zyddnys** [comment]
> > change print to logger.info
> 
> We dont have logging in web_main yet.

sorry  i didn't realize that

**Comment by zyddnys** [comment]
Change to `print('Using cached result for {task_id}')` and i will merge

**Pull request merged**